ccd-definition/FixedLists/OrderType-SDO-nonprod.json

```diff
@@ -2,7 +2,7 @@
   {
     "ID": "OrderType",
     "ListElementCode": "DISPOSAL",
-    "ListElement": "Disposal/Final hearing",
+    "ListElement": "Disposal hearing",
```

Comment:
I think this change is covered by your other PR: https://github.com/hmcts/civil-ccd-definition/pull/661/files

You could close your other PR or remove the change from this one, it's up to you